tools/gen-postmortem-metadata.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+#
+# gen-postmortem-metadata.py output_file.cc
+#
+# Creates debugging symbols to help naviage Node's internals using post-mortem
+# debugging tools.
+#
+
+import sys
+
+
+class DebugSymbol(object):
+  type_ = 'int'
+  _prefix = 'nodedbg_'
+
+  def __init__(self, name, value, headers=[], type_=None):
+    self.name = name
+    self.value = value
+    self.headers = headers
+    self.type_ = type_ or DebugSymbol.type_
+
+  @classmethod
+  def get_headers(cls, debug_symbols):
+    '''
+    Return a list of headers without duplicates, preserving the order they were declared
+    '''
+    seen = set()
+    seen_add = seen.add  # faster than seen.add on each iteration
```

Comment:
It probably makes no measurable difference.  I'd simply drop this and use `seen.add(h)` below.